<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="2" ignored="0" total="6" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="TestNG Learning" duration-ms="176" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
    <groups>
      <group name="high">
        <method signature="Case1.doUserReg()[pri:1, instance:testnglearning.Case1@26aa12dd]" name="doUserReg" class="testnglearning.Case1"/>
        <method signature="Case2.validateTitles()[pri:0, instance:testnglearning.Case2@536aaa8d]" name="validateTitles" class="testnglearning.Case2"/>
        <method signature="Case3.doUserReg()[pri:1, instance:testnglearning.Case3@1877ab81]" name="doUserReg" class="testnglearning.Case3"/>
        <method signature="Case3.doLogin()[pri:2, instance:testnglearning.Case3@1877ab81]" name="doLogin" class="testnglearning.Case3"/>
      </group> <!-- high -->
      <group name="low">
        <method signature="Case1.doLogin()[pri:2, instance:testnglearning.Case1@26aa12dd]" name="doLogin" class="testnglearning.Case1"/>
      </group> <!-- low -->
      <group name="med">
        <method signature="Case3.doUserReg()[pri:1, instance:testnglearning.Case3@1877ab81]" name="doUserReg" class="testnglearning.Case3"/>
        <method signature="Case3.doLogin()[pri:2, instance:testnglearning.Case3@1877ab81]" name="doLogin" class="testnglearning.Case3"/>
        <method signature="Case3.isSkip()[pri:3, instance:testnglearning.Case3@1877ab81]" name="isSkip" class="testnglearning.Case3"/>
      </group> <!-- med -->
    </groups>
    <test name="Login Test" duration-ms="83" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
      <class name="testnglearning.Case1">
        <test-method status="PASS" signature="createDbConn()[pri:0, instance:testnglearning.Case1@26aa12dd]" name="createDbConn" is-config="true" duration-ms="14" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDbConn -->
        <test-method status="PASS" signature="doUserReg()[pri:1, instance:testnglearning.Case1@26aa12dd]" name="doUserReg" duration-ms="10" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:testnglearning.Case1@26aa12dd]" name="launchBrowser" is-config="true" duration-ms="0" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="quitBrowser()[pri:0, instance:testnglearning.Case1@26aa12dd]" name="quitBrowser" is-config="true" duration-ms="1" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:testnglearning.Case1@26aa12dd]" name="launchBrowser" is-config="true" duration-ms="7" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="doLogin()[pri:2, instance:testnglearning.Case1@26aa12dd]" name="doLogin" duration-ms="4" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="quitBrowser()[pri:0, instance:testnglearning.Case1@26aa12dd]" name="quitBrowser" is-config="true" duration-ms="1" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
        <test-method status="PASS" signature="closeDbConn()[pri:0, instance:testnglearning.Case1@26aa12dd]" name="closeDbConn" is-config="true" duration-ms="0" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDbConn -->
      </class> <!-- testnglearning.Case1 -->
    </test> <!-- Login Test -->
    <test name="Validate Title Test" duration-ms="17" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
      <class name="testnglearning.Case2">
        <test-method status="FAIL" signature="validateTitles()[pri:0, instance:testnglearning.Case2@536aaa8d]" name="validateTitles" duration-ms="7" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Yahoo.com] but found [Gmail.com],
Element not found expected [true] but found [false],
Test failed as the condition is not met]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Yahoo.com] but found [Gmail.com],
Element not found expected [true] but found [false],
Test failed as the condition is not met
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at testnglearning.Case2.validateTitles(Case2.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTitles -->
      </class> <!-- testnglearning.Case2 -->
    </test> <!-- Validate Title Test -->
    <test name="Dependency Test" duration-ms="25" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
      <class name="testnglearning.Case3">
        <test-method status="FAIL" signature="doUserReg()[pri:1, instance:testnglearning.Case3@1877ab81]" name="doUserReg" duration-ms="2" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User not registered successfully]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User not registered successfully
at org.testng.Assert.fail(Assert.java:96)
at testnglearning.Case3.doUserReg(Case3.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method status="SKIP" signature="doLogin()[pri:2, instance:testnglearning.Case3@1877ab81]" name="doLogin" duration-ms="0" started-at="2019-04-14T06:21:16Z" depends-on-methods="testnglearning.Case3.doUserReg" finished-at="2019-04-14T06:21:16Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Case3.doLogin()[pri:2, instance:testnglearning.Case3@1877ab81] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Case3.doLogin()[pri:2, instance:testnglearning.Case3@1877ab81] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="SKIP" signature="isSkip()[pri:3, instance:testnglearning.Case3@1877ab81]" name="isSkip" duration-ms="3" started-at="2019-04-14T06:21:16Z" finished-at="2019-04-14T06:21:16Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test as the condition is not met]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test as the condition is not met
at testnglearning.Case3.isSkip(Case3.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
      </class> <!-- testnglearning.Case3 -->
    </test> <!-- Dependency Test -->
  </suite> <!-- TestNG Learning -->
</testng-results>
